src/guiguts/application.py

```diff
@@ -242,8 +242,8 @@ def init_edit_menu(self):
     def init_view_menu(self):
         """Create the View menu."""
         menu_view = Menu(menubar(), "~View")
-        menu_view.add_button("~Dock", self.mainwindow.dock_image)
-        menu_view.add_button("~Float", self.mainwindow.float_image)
+        menu_view.add_button("~Dock", self.mainwindow.dock_image, "Cmd/Ctrl+X")
+        menu_view.add_button("~Float", self.mainwindow.float_image, "Cmd/Ctrl+Y")
         menu_view.add_button("~Load Image", self.mainwindow.load_image)
```

Comment:
Please could you change this one to use "T" instead of "Y" - just to test out what it's like to get changes suggested during code review. Will message further in Slack...

Reply:
Made that change, Did push to origin.